client/src/components/ui/forms/staff/RemoveStaff.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+    import { Staff } from '../../../../api/staff.ts'; 
+    import { PersonPayload } from '../../../types.ts';
+    import { staffList } from '../../../../pages/dashboard/stores/StaffStore.ts';
+    
+    import Button from '../../Button.svelte';
+    import PersonDelete from '../../../icons/PersonDelete.svelte';
+    import { ButtonType, IconColor } from '../../../types.ts';
+
+    export let payload: PersonPayload;
+
+    async function handleRemove() {
+        try{
+            await Staff.remove({
+                office: payload.office,
+                user_id: payload.id,
+            });
+            await staffList.reload?.();
+        } catch (err) {
+            // TODO: No permission handler
+            alert(err);
+        }
+    }
+</script>
+
+
+<p>Are you sure you want to remove {payload.email} from {payload.office}?</p>
```

Comment:
```suggestion
<p>Are you sure you want to remove {payload.email} from {officeName}?</p>
```
I recommend replacing the office ID (which is payload.office) with the actual Office name (string) to make it more user friendly, 
`$: officeName = $allOffices[office] ?? 'No office name.';`